lib/fastlane/plugin/wpmreleasetoolkit/helper/promo_screenshots_helper.rb

```diff
@@ -32,17 +32,62 @@ def initialize
         UI.user_error!('`drawText` not found – install it using `brew install automattic/build-tools/drawText`.') unless system('command -v drawText')
       end
 
-      def read_json(configFilePath)
+      def read_config(configFilePath)
         configFilePath = resolve_path(configFilePath)
 
         begin
-          return JSON.parse(open(configFilePath).read)
-        rescue
-          linter = JsonLint::Linter.new
-          linter.check(configFilePath)
-          linter.display_errors
```

Comment:
I considered replacing the `jsonlint` gem with `yamllint` gem, but then found out that `yamllint` is not maintained anymore and trying to `bundle install` it leads to a deprecation warning about it relying on an old `trollop` gem.

I figured that since this was the only place where we used `JsonLint` and linting, and given the config files are not that often modified anyway, and if they are it would likely be with a dedicated IDE, and the most likely mistakes won't be syntax errors in the YAML but incorrect keys or structure used compared to what's expected… so that wasn't worth the effort to keep a linter after all.